### PR TITLE
CyberArk 6JUL

### PR DESCRIPTION
**Pull request by JimmyJamCABD**
Attempting from GitHubDesktop

Fixes:
Made changes requested from PR 813

### CODE REVIEW

**Comment by JimmyJamCABD** [comment]
Just wanted to bring this to the top of your inbox :-)

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -797,5 +797,20 @@
     "templateRelativePath": "SysmonThreatHunting.json",
     "subtitle": "",
     "provider": "Azure Sentinel community"
+  },
+  {
+  "workbookKey": "CyberArkWorkbook",
+  "logoFileName": "CyberArk_Logo.svg",
```
Comment (preetikr) [inline Workbooks/WorkbooksMetadata.json:803]: Please check in this logo file in the images -> logo folder per the workbook guidance https://aka.ms/sentinelgithubworkbooks

File: Workbooks/WorkbooksMetadata.json [hunk 2]
```diff
@@ -797,5 +797,20 @@
...
+  "description": "The CyberArk Syslog connector allows you to easily connect all your CyberArk security solution logs with your Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. Integration between CyberArk and Azure Sentinel makes use of the CEF Data Connector to properly parse and display CyberArk Syslog messages.",
+  "dataTypesDependencies": [ "CommonSecurityLog" ],
+  "dataConnectorsDependencies": [ "CyberArk" ],
+  "previewImagesFileNames": [ "CyberArk_White1.png", "CyberArk_Black1.png" ],
```
Comment (preetikr) [inline Workbooks/WorkbooksMetadata.json:807]: Please check in the two preview files in the images folder per the workbook guidance https://aka.ms/sentinelgithubworkbooks

File: DataConnectors/CyberArk Data Connector.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (preetikr) [inline DataConnectors/CyberArk Data Connector.json:1]: Please check in the logo file for data connectors per step 5 of [data connectors guidance](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector)

**Pull request closed without merging**